develop/parachain-devs/get-started/polkadot-sdk/get-started-psdk.md

```diff
@@ -0,0 +1,137 @@
+        - [`frame`](https://github.com/paritytech/polkadot-sdk/tree/master/substrate/frame){target=\_blank} - The FRAME development environment provides modules (called "pallets") and support libraries that you can use, modify, and extend to build runtime logic to suit the needs of your blockchain. Includes key pallets like `BABE` and `GRANDPA`
+
+    === "Resources"
+
+        - Read about the [Substrate module](https://paritytech.github.io/polkadot-sdk/master/polkadot_sdk_docs/polkadot_sdk/substrate/index.html#){target=\_blank}
+        - [Substrate Stack Exchange](https://substrate.stackexchange.com/){target=\_blank} - ask questions and get answers from other developers
+
+??? function "[**Cumulus**](https://github.com/paritytech/polkadot-sdk/tree/master/cumulus){target=\_blank} - a set of libraries and tools focused on distribution, p2p, database, and synchronization layers for Polkadot parachains"
+
+    === "Key Features"
+
+        - [`parachain-consensus`](https://github.com/paritytech/polkadot-sdk/blob/master/cumulus/client/consensus/common/src/parachain_consensus.rs){target=\_blank} - a consensus engine to run a Polkadot node internally and dictate to the client and synchronization algorithms which chain to follow, finalize, and treat as best
+        - [`polkadot-parachain`](https://github.com/paritytech/polkadot-sdk/tree/master/cumulus/polkadot-parachain){target=\_blank} - binary which implements a Polkadot collator for the parachain  
+        - [`validate_block`](https://github.com/paritytech/polkadot-sdk/tree/1617852a2f1caec796f2b218aecae2facaeacad6/cumulus/pallets/parachain-system/src/validate_block){target=\_blank} - function that is expected by Polkadot to validate a state transition
+
+    === "Resources"
+
+        - Read the [Cumulus Overview](https://github.com/paritytech/polkadot-sdk/blob/master/cumulus/docs/overview.md){target=\_blank}
+        - [Cumulus installation and setup](https://github.com/paritytech/polkadot-sdk/tree/master/cumulus#installation-and-setup){target=\_blank} - instructions to prepare your developer environment and launch a local network
+
+For an in-depth dive into the monorepo, the [Polkadot SDK Rust documentation](https://paritytech.github.io/polkadot-sdk/master/polkadot_sdk_docs/polkadot_sdk/index.html){target=\_blank} is highly recommended
+
+<!-- TODO: Glossary link ParaId -->
+
+## Zero to Hero with The Polkadot SDK
+
+The Polkadot SDK is a versatile tool that can be used to build both standalone blockchains and parachains. For standalone blockchains, developers can leverage the SDK's components to create independent networks with custom functionality. The process for deploying a parachain on a Polkadot [core](https://wiki.polkadot.network/docs/learn-agile-coretime#core){target=\_blank} can be summarized as follows:
```

Comment:
I don't know if this section makes sense. No one who is trying to get what the Polkadot SDK is will understand what all of these steps mean

Reply:
Yeah, I'm happy to remove this section. Will refocus it.